### PR TITLE
6.5.0: View data in web UI

### LINKED ISSUE #3329: View Data option in web UI
### Description
Customers can retrieve resource definitions in the web UI (View Data option) starting with Sensu Go 6.5.0.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/6.5/web-ui/view-manage-resources/
Link to ^ from pages that explain how to retrieve resource definitions

### Corrections needed
Create a new section for screenshot and explanation of View Data feature in web UI docs.

Add link to web UI doc section throughout docs where we explain how to retrieve resource definitions (in various guides, for example).

### Additional context
Issue: https://github.com/sensu/sensu-enterprise-go/issues/1220
PR: https://github.com/sensu/sensu-enterprise-go/pull/1633

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Describes how to view resource data (with animated screenshot) in the web UI docs
- Adds tips with links to the web UI doc ("...you can also view resource definitions in the web UI...") throughout the docs that include sensuctl <resource> info commands

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/3329

### CODE REVIEW

File: content/sensu-go/6.5/next-steps.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ In the quick start guide, you used sensuctl for ad hoc configuration when you ad
 But Sensu resources like checks and events also have detailed definitions that you can store, maintain, and reuse just like you would store, maintain, and reuse code.
 Combining Sensu’s flexible APIs with this monitoring as code approach allows you to scale everything we’re describing on this page.
 
-Here’s how to retrieve the resource definition for your NTP check and copy it to a YAML or JSON file.
+Here’s how to use sensuctl retrieve the resource definition for your NTP check and copy it to a YAML or JSON file.
```
Comment (dabria) [inline content/sensu-go/6.5/next-steps.md]: > Here’s how to use sensuctl retrieve

I think you might need `how to use sensuctl to retrieve`

**Review by dabria — APPROVED** [review]
Looks good! Just one grammar thing (maybe)

File: content/sensu-go/6.5/next-steps.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/6.5/next-steps.md]: yes, I did mean that! Thank you!

Comment (hillaryfraley) [inline content/sensu-go/6.5/next-steps.md]:
```suggestion
Here’s how to use sensuctl to retrieve the resource definition for your NTP check and copy it to a YAML or JSON file.
```

**Pull request merged**